src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -653,6 +658,10 @@ protected DynamoModel(IStartConfiguration config)
                 PreferenceSettings.PropertyChanged += PreferenceSettings_PropertyChanged;
             }
 
+            //Due that the Expanders settings are stored by session (when dynamo is closed, settings are lost) then we read the settings from a json string
+            string jsonExpandersSettings = Resources.PreferencesExpandersSettings;
```

Comment:
this seems quite odd - usually in (our repos anyway) resx files are intended for localized content - not data like this.

Why use json at all, why not just have the default constructor set these default values?

Reply:
@mjkkirschner 
I did the change so instead of using the json entry I'm initializing the List in the Constructor, see commit: https://github.com/DynamoDS/Dynamo/pull/11689/commits/af8e31e76e11ad15e55a50a7526c57c093df3d6e